### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

Add EOSC-hub funding acknowledgement as requested by project

----

**Related issue :**

**Consider author for inclusion**:

- [ ] author list
- [ ] contributor list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,13 @@ Currently supported cloud middleware:
 * OpenStack
 * OpenStack/ooi
 
+## Acknowledgement
+
+This work is co-funded by the [EOSC-hub project](http://eosc-hub.eu/) (Horizon 2020) under Grant number 777536.
+<img src="https://wiki.eosc-hub.eu/download/attachments/1867786/eu%20logo.jpeg?version=1&modificationDate=1459256840098&api=v2" height="24">
```
Comment (gwarf) [inline README.md]: Please use Markdown tags not raw HTML

Comment (enolfc) [reply README.md]: but then I cannot specify height...

Comment (gwarf) [reply README.md]: Arf, I really don't like this much it looks quite ugly.
But as far as I can see there aren't more alternatives apart maybe embedding/hosting an image at the right side.
So ok let's merge it to not loose to much time with this.

**Comment by gwarf** [comment]
I've merged even if tests are breaking for python 3.4 and working for 2.7, as it's not related to this commit

**Pull request merged**